docs/resources/resource.md

```diff
@@ -55,13 +64,24 @@ resource "twingate_resource" "resource" {
 
 ### Optional
 
+- `access` (Block List, Max: 1) Restrict access to certain groups or service accounts (see [below for nested schema](#nestedblock--access))
 - `group_ids` (Set of String, Deprecated) List of Group IDs that have permission to access the Resource, cannot be generated by Terraform and must be retrieved from the Twingate Admin Console or API
 - `protocols` (Block List, Max: 1) Restrict access to certain protocols and ports. By default or when this argument is not defined, there is no restriction, and all protocols and ports are allowed. (see [below for nested schema](#nestedblock--protocols))
 
 ### Read-Only
 
 - `id` (String) Autogenerated ID of the Resource, encoded in base64
 
+<a id="nestedblock--access"></a>
+### Nested Schema for `access`
+
+Optional:
+
+- `group_ids` (Set of String) List of Group IDs that have permission to access the Resource.
+- `non_authoritative` (Boolean) Determines authoritative behaviour for handling the Resource's groups or service accounts.
```

Comment:
I'm curious how this would work being assigned to an access block instead of to the resource itself.  It seems this should be set at the resource level, because eventually this will support multiple access blocks and it doesn't make sense for some to be "authoritative" and others to not be.  But maybe I'm mis-understanding some use case?

Reply:
I think multiple "authoritative" access blocks will make conflicts with each other, multiple non-authoritative blocks can be merged into one.

Currently put "authoritative" flag inside access block, as it's logically relates more to access settings than to resource itself. 

I guess multiple access blocks would make more sense with policies.

Reply:
I see that @alexmensch had originally specified it as part of the `access {}` block in the original issue.  I'm curious if that was just a mistake or if it was on purpose.  I don't see how it would work once you support multiple access blocks.

Reply:
Hey @tjstansell we've clarified this behavior here: https://github.com/Twingate/terraform-provider-twingate/issues/245#issuecomment-1386003243

Bit of internal back and forth missed communication in the rush before the holiday break. The whole team is back now, so we're back at it! :)